components/builder-minio/habitat/hooks/init

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+exec 2>&1
+
+{{#if cfg.use_ssl}}
+    mkdir -p {{pkg.svc_config_path}}/certs
+    cp {{pkg.svc_files_path}}/private.key {{pkg.svc_config_path}}/certs/private.key
```

Comment:
By switching these to `cp` from `ln` we'll get a loop on init failure until the files get uploaded to the service group which is exactly what we want here.

Reply:
👍